src/components/tables/DataTable.vue

```diff
@@ -137,7 +95,7 @@ export interface Props<T, K extends keyof T> {
   /**
    * attribute to use from column definitions to display column titles
    */
-  columnAttr?: string;
+  columnAttr?: keyof TableColumn<T>;
```

Comment:
Not affecting this PR but do we actually use this anywhere? @tewshi @lukicenturi 
Don't we just always use the default value which is `label`, maybe we can get rid of it in the future unless you can think of a use.

Reply:
noted.